package sec02_exceptionhandlingsyntax.EX05;

public class TryWithResource_1 {
	public static void main(String[] args) {
		
		//# 자동리소스 해제
//		try() {
//			
//		}
//		catch() {
//			
//		}
	}
}
